test/unit/js/GcsPersistorTests.js

```diff
@@ -44,6 +44,7 @@ describe('GcsPersistorTests', function() {
           user_files: 'user_files'
         },
         gcs: {
+          deleteConcurrency: 1,
```

Comment:
Did this need to go in here? It will disable concurrency on the delete-directory test.

Reply:
i added this because of some error that was coming up in unit testing, but it must have been caused by something else I fixed, because the tests passed once I took this back out.